dev_docs/HOW_TO.md

```diff
@@ -11,14 +11,17 @@ Adding a new intake field requires changes to both apm-data and apm-server repos
    - RUM v3: [input/elasticapm/internal/modeldecoder/rumv3/model.go](/input/elasticapm/internal/modeldecoder/rumv3/model.go)
 2. Run `make generate` to generate the corresponding `model_generated.go` and JSON specification from the modified `model.go` in step 1.
 3. Run `make update-licenses` to add license header to `model_generated.go` generated in step 2.
-4. Add the new field to the corresponding file in `model/`, e.g. `span.go`.
-   1. Add the field to the struct.
-   2. Modify the `fields` function to include the field.
+4. Add the new field to the corresponding file in `model/proto/`, e.g. `span.proto`.
+   1. Run `make generate-modelpb` to generate the corresponding `span.pb.go`.
+   2. Run `make update-licenses fmt` to add license header to `span.pb.go` generated in step 1.
```

Comment:
A little bit of ambiguity here, there's also a step 1 in the outer layer
```suggestion
   2. Run `make update-licenses fmt` to add license header to `span.pb.go` generated in the previous step.
```

Reply:
Thanks! Good catch :+1: